rector.php

```diff
@@ -28,8 +28,6 @@
     $parameters->set(Option::FILE_EXTENSIONS, ['php', 'module', 'theme', 'install', 'profile', 'inc', 'engine']);
     $parameters->set(Option::AUTO_IMPORT_NAMES, true);
     $parameters->set(Option::IMPORT_SHORT_CLASSES, false);
-    // @phpstan-ignore-next-line
-    $parameters->set(Option::IMPORT_DOC_BLOCKS, false);
```

Comment:
removed in 0.12.21: https://github.com/rectorphp/rector-src/commit/c35cc97d381f97c602cccb86ccf1d688a869cbdd